module/pom.xml

```diff
@@ -256,7 +285,15 @@
       <version>${mule.version}</version>
       <scope>test</scope>
     </dependency>
-
+    <dependency>
+      <groupId>org.mule.sdk</groupId>
+      <artifactId>mule-sdk-api</artifactId>
+      <version>0.7.0-rc12</version>
```

Comment:
there is a final version for this, we should use `0.7.0` (again, as a property).

Reply:
changed